samples/amd-webpack-react/src/index.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+
+const schema = {
+    Plugins: [],
+    Databases: {
+        Samples: {
+            Name: 'Samples',
+            Tables: {
+                StormEvents: {
+                    Name: 'StormEvents',
+                    DocString:
+                        'A dummy description to test that docstring shows as expected when hovering over a table',
+                    OrderedColumns: [
+                        {
+                            Name: 'StartTime',
+                            Type: 'System.DateTime',
+                            CslType: 'datetime',
+                            DocString: 'The start time',
+                        },
+                        {
+                            Name: 'State',
+                            Type: 'System.String',
+                            CslType: 'string',
+                        },
+                    ],
+                },
+            },
+            Functions: {},
+        },
+    },
+};
+
+function App() {
+    const divRef = React.useRef(null);
+
+    React.useLayoutEffect(() => {
+        let disposed = false;
+        let editor;
+        // https://webpack.js.org/api/module-variables/#__non_webpack_require__-webpack-specific
```

Comment:
I'd maybe add a comment here why we need this special `__non_webpack_require__()`? Or if in the context of an AMD module that this makes sense inherently then this link is enough.